arangodb/content.md

```diff
@@ -4,9 +4,7 @@ ArangoDB is a multi-model, open-source database with flexible data models for do
 
 The supported data models can be mixed in queries and allow ArangoDB to be the aggregation point for the data request you have in mind.
 
-> [arangodb.com](https://arangodb.com)
-
-%%LOGO%%
```

Comment:
If you drop this then the logo will no longer be in the generated [`README.md`](https://github.com/docker-library/docs/blob/master/arangodb/README.md).